packages/connectors/connector-dingtalk-web/README.md

```diff
@@ -0,0 +1,39 @@
+# 钉钉网页
+
+先进企业协作与管理平台，一站式无缝办公协作，团队上下对齐目标，全面激活组织和个人。
+
+## 开始上手
+
+钉钉网页连接器是为桌面网页应用设计的。它采用了 OAuth 2.0 认证流程。
+
+## 注册钉钉开发者账号
+
+如果你还没有钉钉开发者账号，请在 [钉钉开放平台](https://open.dingtalk.com) 注册。
+
+## 创建应用
+
+1. 在 [钉钉开发者后台](https://open-dev.dingtalk.com/console/index) 中，点击「创建应用」
```

Comment:
Can we also provide an English version of the README? Like what we have for Aliyun and WeChat connectors?
Thanks.

Reply:
> Can we also provide an English version of the README? Like what we have for Aliyun and WeChat connectors? Thanks.

ok. i updated the docs

Reply:
Thank you! I appreciate it.